content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Data Services on the HPE GreenLake platform
+version: 0.1.1
+description: The landing page for all Storage BU related SaaS and cloud services
+  with future unified API in the plan
+image: /img/greenlake/dscc-icon-transparent.png
+width: large
+priority: 1
+active: false
+tags:
+  - hpe-greenlake
+  - api
+  - data-services-for-hpe-greenlake
+---
+<style>
+ul li{
+ font-size:27px;
+}
+</style>
+
+<style>
+ol li{
+ font-size:27px;
+}
+</style>
+
+Data Services on the HPE GreenLake platform is a group of services which are part of the service catalogues in the HPE GreenLake Edge to Cloud Platform. These services bring the cloud experiences to HPE GreenLake customers, wherever the data lives, across on-premises or public cloud throughout its lifecycle. With the streamlined ordering, provisioning, management, protection, analysis, and archiving, customers can achieve higher agility to innovate using latest trend in I.T. such as artificial intelligence. For more information on how to leverage these Data Services on the HPE GreenLake platform, please visit HPE Data Management [website](https://www.hpe.com/us/en/storage/data-services-cloud-console.html).
```

Comment:
Naming convention for HPE GreenLake edge to cloud platform is: HPE Greenlake edge-to-cloud platform.

Reply:
We need to make sure we follow the latest name rules for HP Greenlake. It's changing AGAIN

Reply:
@rondharma 
We recently received the information below for Discover (From Kailah Findensen):

--> Request to move forward is to drop “edge-to-cloud platform” or “platform” in any reference to GreenLake. So at this point we should only be saying “HPE GreenLake”

However, Paul Zinn shared the following information:
this guidance is early, as the full decision on how to refer to HPE GreenLake and the platform hasn’t been made yet. I would recommend holding off on implementing Kailah’s ask for now so that you don’t end up making two passes through your material instead of one. When I hear of the final decision, I’ll pass it on.
 

So, until further notice, let's stick with the current naming convention:
HPE GreenLake edge-to-cloud platform or HPE GreenLake platform

Sorry for any confusion this might create.

Thanks
Denis

Reply:
Thanks Denis, Didier